core/scripts/Voting.js

```diff
@@ -1,9 +1,52 @@
 const Voting = artifacts.require("Voting");
 const { VotePhasesEnum } = require("../utils/Enums");
+const fetch = require("node-fetch");
+
+//require("dotenv").config();
+// Get API keys from environment variables or the `.env` file.
+// const cryptocompareKey = process.env.CRYPTO_COMPARE_KEY;
+
+function stripApiKey(str, key) {
+  return str.replace(key, "{redacted}");
+}
+
+// Gets JSON from a URL or throws.
+const getJson = async url => {
+  const response = await fetch(url);
+  const json = await response.json();
+  if (!json) {
+    throw `Query [${url}] failed to get JSON`;
+  }
+  return json;
+};
 
-// TODO(#492): Implement price fetching logic.
 async function fetchPrice(request) {
-  return web3.utils.toWei("1.5");
+  const identifier = request.identifier;
```

Comment:
As you've run in to, the tests will unfortunately fail because they'll try to look up a price, so we'll need a way to exclude this function from the test.

Also the API here is a little different: `request.identifier` is a string, e.g., "BTCUSD".

Here's what I recommend:
* Leave `fetchPrice` as it is. We'll have to figure out the right strategy for excluding this code from the test flow of VotingSystem, let's chat about that separately.
* Move this logic into a new function `fetchCryptoComparePrice(fsym, tsym)`.

In your next PR, let's define a mapping from "BTCUSD" to `fsym` and `tsym`. It could be as simple as:
`{ "BTCUSD": { source: "CryptoCompare", fsym: "BTC", tsym: "USD" }`.

Then we can fill in `fetchPrice` to call `fetchCryptoComparePrice` with the right arguments.

Thoughts?